src/doom/s_sound.c

```diff
@@ -445,6 +445,40 @@ void S_StartSound(void *origin_p, int sfx_id)
         I_Error("Bad sfx #: %d", sfx_id);
     }
 
+    // substitute missing sounds in Doom 1.2
+    if (gameversion <= exe_doom_1_2)
+    {
+        switch(sfx_id)
+        {
+            case sfx_bdcls:
```

Comment:
Blazing doors weren't added until Doom II (first version being 1.666) so I don't know why this is necessary.

Reply:
Removed in https://github.com/chocolate-doom/chocolate-doom/commit/77f49e5ea5aae45fe904306a9364018c0fc23b71 .